### PR TITLE
WIP - Telegram notifications

### PR DESCRIPTION
**Pull request by shendel**

### CODE REVIEW

**Comment by shendel** [comment]
Work in process, dont merge

File: resources/views/users/subscriptions-table.blade.php [hunk 1]
```diff
@@ -12,12 +12,14 @@
...
     </tr>
     <tr>
         <td>{{ __('There is a new conversation') }}</td>
         <td class="subscriptions-email"><input type="checkbox" @include('users/is_subscribed', ['medium' => App\Subscription::MEDIUM_EMAIL, 'event' => App\Subscription::EVENT_NEW_CONVERSATION]) name="{{ $subscriptions_formname }}[{{ App\Subscription::MEDIUM_EMAIL }}][]" value="{{ App\Subscription::EVENT_NEW_CONVERSATION }}"></td>
         <td class="subscriptions-browser"><input type="checkbox" @include('users/is_subscribed', ['medium' => App\Subscription::MEDIUM_BROWSER, 'event' => App\Subscription::EVENT_NEW_CONVERSATION]) name="{{ $subscriptions_formname }}[{{ App\Subscription::MEDIUM_BROWSER }}][]" value="{{ App\Subscription::EVENT_NEW_CONVERSATION }}"></td>
+        <td class="subscriptions-telegram"><input type="checkbox" @include('users/is_subscribed', ['medium' => App\Subscription::MEDIUM_TELEGRAM, 'event' => App\Subscription::EVENT_NEW_CONVERSATION]) name="{{ $subscriptions_formname }}[{{ App\Subscription::MEDIUM_TELEGRAM }}][]" value="{{ App\Subscription::EVENT_NEW_CONVERSATION }}"></td>
```
Comment (shendel) [inline resources/views/users/subscriptions-table.blade.php:22]: Need code-health - email, browser, telegram, mobile has own code

**Comment by freescout-helpdesk** [comment]
Thanks. But not needed in the core. Feel free to implement as a [custom module](https://github.com/freescout-helpdesk/freescout/wiki/Modules-Development) and share: https://github.com/freescout-helpdesk/freescout/wiki/Community-Modules

**Comment by freescout-helpdesk** [comment]
Also there is already: https://freescout.net/module/telegram/

**Pull request closed without merging**